Sample Data/ASIM/Citrix_NetScaler_ASimWebSession_IngestedLogs.csv

```diff
@@ -0,0 +1,24 @@
+1a0e2567-2e58-4989-ad18-206108185325,"8/24/2023, 1:06:13 PM",Citrix,NetScaler,NS11.0,APPFW,APPFW_SQL,6,,geolocation=Unknown;method=POST,blocked,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,SQL Keyword check failed for field text_area=('),,,,,,,,,,,,,,http://aaron.stratum8.net/FFC/login_post.php,,,,,,,,,,,,,,,,9459,10.217.253.62,,,,,,,,,,,,,,,,,,,,,,,78,,,834,,,,,pr_ffc,,PPE1,,eVJMMPtZ2XgylGrHjkx3rZLfBCI0002,,ALERT,,2015,,,,,,,,,,,,,,,,,,,,,,,,,,,,,localhost,OpsManager,blocked,,CommonSecurityLog,,b9b77a83-419e-11ee-94cf-002248291ba6
```

Comment:
These logs are from sample available on website - https://docs.netscaler.com/en-us/citrix-adc/current-release/application-firewall/logs.html

Did you create test environment for this parser or this is built based on sample logs you found over the internet?

Reply:
We have set up a test environment but few of the logs are generated from it so logs are the combination of Sample + live data.